Fluid/Filters/MiscFilters.cs

```diff
@@ -100,7 +101,37 @@ public static FilterCollection WithMiscFilters(this FilterCollection filters)
         /// </summary>
         public static ValueTask<FluidValue> Handleize(FluidValue input, FilterArguments arguments, TemplateContext context)
         {
-            return new StringValue(HtmlCaseRegex.Replace(input.ToStringValue(), "-$1$2").ToLowerInvariant());
+            var value = HtmlCaseRegex
```

Comment:
Is this regex required?

Reply:
Yep, this is what you did before to convert into Kebab-Case, but we could remove after this PR. For now it's fine to keep it, because it reduces a lot of code ;)

One comment about that `RegEx`, can we set its options into compiled?

Reply:
Pretty sure it could be removed with what you have written.